src/org/parosproxy/paros/CommandLine.java

```diff
@@ -328,6 +332,28 @@ private boolean parseKeywords(String[] args, int i) throws Exception {
                 this.configs.put(pair.substring(0, eqIndex), pair.substring(eqIndex + 1));
                 result = true;
             }
+        } else if (checkPair(args, CONFIG_FILE, i)) {
+            String conf = keywords.get(CONFIG_FILE);
+            File confFile = new File(conf);
+            if (! confFile.exists()) {

```

Comment:
Yes, isFile (in addition to canRead) would be better :)
btw thanks for jumping in @Grunny - the more eyes we have on the code the better!